### ISSUE TITLE
Add mega cloud drive support for vault sync.

### ISSUE THREAD

**Issue by nikunjkumarnakum** [buttercup/dossier#28]
Please consider adding mega drive support for vault sync like it has for dropbox.

**Comment by Alfagiovanni56** [comment]
I would also be very interested in adding Mega for vault sync!! I just dropped Dropbox for Mega, which gives much better support to opensource and esp. linux environments, beside that it is a zero-knowledge cloud service, of course.

**Comment by visionintruetype** [comment]
would love to see support for mega!

**Comment by visionintruetype** [comment]
would love mega support! options are good!!

**Comment by Alfagiovanni56** [comment]
Yes, of course. I would be glad if it could be added. It would guarantee full functionality on all devices (computer, mobile phone...) without work-arounds. Compared to Google Drive and Dropbox Mega has the benefit of being end-to-end encrypted, which makes it by far a better solution for managing passwords.

**Comment by perry-mitchell** [comment]
I'd accept PRs for this. Needs to be added in core, desktop and mobile. I'm happy to guide anyone attempting this, but in depth TS/JS knowledge is required.

I won't have time to add this personally.
